sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CreationData.cs

```diff
@@ -42,13 +42,22 @@ public CreationData()
         /// of a blob to be imported into a managed disk.</param>
         /// <param name="sourceResourceId">If createOption is Copy, this is the
         /// ARM id of the source snapshot or disk.</param>
-        public CreationData(string createOption, string storageAccountId = default(string), ImageDiskReference imageReference = default(ImageDiskReference), string sourceUri = default(string), string sourceResourceId = default(string))
+        /// <param name="sourceUniqueId">If this field is set, this is the
+        /// unique id identifying the source of this resource.</param>
+        /// <param name="uploadSizeBytes">If createOption is Upload, this is
+        /// the size of the contents of the upload including the VHD footer.
+        /// This value should be between 1073741824 (1 GiB + 512 bytes for the
```

Comment:
This does not look correct. We should be allowing disks of size > 10 MB or so. Please check.

Reply:
I thought empty disks and upload disks have a minimum of 1gb?

Reply:
PR to fix the description:
https://github.com/Azure/azure-rest-api-specs/pull/7025/commits/9d5e30df16e3ea43cb3b21b5f5264373d8ed2397